python/GafferUITest/BackgroundMethodTest.py

```diff
@@ -107,23 +107,25 @@ def __setTextPlug( self ) :
 
 			return self.__script["n"]["sum"]
 
-	class WaitingSlot( GafferTest.CapturingSlot ) :
+	class __SetValueWaitingSlot( GafferTest.CapturingSlot ) :
 
-		def __init__( self, signal ) :
+		def __init__( self, valueWidget ) :
 
-			GafferTest.CapturingSlot.__init__( self, signal )
+			GafferTest.CapturingSlot.__init__( self, valueWidget.valueChangedSignal() )
+			self.__args = args
```

Comment:
Well, this LGTM, but it didn't LGTCI :

```
======================================================================
Error: ERROR: test (GafferUITest.BackgroundMethodTest.BackgroundMethodTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/gaffer/gaffer/build/python/GafferUITest/BackgroundMethodTest.py", line 137, in test
    ws = self.__SetValueWaitingSlot( w )
  File "/__w/gaffer/gaffer/build/python/GafferUITest/BackgroundMethodTest.py", line 115, in __init__
    self.__args = args
NameError: global name 'args' is not defined
```

Pro tip : run the tests before pushing ;)

Reply:
Haha, sorry I _had_ run them, then thought I'd left a stray args in the signature from during our chat. doh! That'll teach me! More than just a stray :)

Reply:
Third time lucky. Sorry about that.